app/components/work_history_item_component.rb

```diff
@@ -38,13 +38,29 @@ def organisation
   attr_accessor :item
 
   def formatted_start_date
-    item.start_date.to_s(:month_and_year)
+    if FeatureFlag.active?(:restructured_work_history)
+      if item.is_a? ApplicationWorkExperience
+        item.start_date_unknown ? item.start_date.to_s(:month_and_year) + ' (approximate)' : item.start_date.to_s(:month_and_year)
+      else
+        item.start_date.to_s(:month_and_year)
+      end
+    else
+      item.start_date.to_s(:month_and_year)
+    end
```

Comment:
Consider simplifying the condition and also use interpolaration to construct the string.

```suggestion
    if FeatureFlag.active?(:restructured_work_history) 
      if item.is_a?(ApplicationWorkExperience) && item.start_date_unknown 
         return "#{item.start_date.to_s(:month_and_year)} (approximate)" 
       end
    end
    
    item.start_date.to_s(:month_and_year)
```